### ISSUE TITLE
Update vignettes

### ISSUE THREAD

**Issue by ALShum** [ijlyttle/shinypod#5]
Update vignettes to reflect the fact that the shinyjs hide/show should be in the presentation layer

**Comment by ijlyttle** [comment]
Yes - have to do this and update the function documentation too.
